projects/vdk-plugins/vdk-meta-jobs/tests/test_meta_job.py

```diff
@@ -303,3 +303,36 @@ def test_meta_job_circular_dependency(httpserver: PluginHTTPServer):
         # no other request should be tried as the meta job fails
         assert isinstance(result.exception, UserCodeError)
         assert len(httpserver.log) == 0
+
+
+def test_meta_job_exceed_limit(httpserver: PluginHTTPServer):
+    jobs = [("job" + str(i), [200], "succeeded") for i in range(1, 8)]
+    api_url = _prepare(httpserver, jobs)
+
+    with mock.patch.dict(
+        os.environ,
+        {
+            "VDK_CONTROL_SERVICE_REST_API_URL": api_url,
```

Comment:
Because we have delayed jobs, you may want to speed up the test by lowering VDK_META_JOBS_DELAYED_JOBS_MIN_DELAY_SECONDS and VDK_META_JOBS_DELAYED_JOBS_RANDOMIZED_ADDED_DELAY_SECONDS to 0 or 1 so we do not wait and re-try almost immediately.

Reply:
Done. Thank you.